tests/Microsoft.Identity.Test.Unit/PublicApiTests/MtlsPopTests.cs

```diff
@@ -456,10 +456,13 @@ public async Task MtlsPop_NonAadAuthorityAsync(string authorityUrl, string autho
         }
 
         [DataTestMethod]
-        [DataRow("https://login.microsoftonline.com", "Public Cloud")]
-        [DataRow("https://login.microsoftonline.us", "Azure Government")]
-        [DataRow("https://login.partner.microsoftonline.cn", "Azure China")]
-        public async Task MtlsPop_WithCommonAsync(string authorityUrl, string cloudType)
+        [DataRow("https://login.microsoftonline.com", TestConstants.Common, "Public Cloud")]
```

Comment:
Do we already have test coverage for, 

```
[DataRow(AuthorityType.Generic, "https://example.com")]
[DataRow(AuthorityType.Adfs, "https://example.com/adfs")]
```

Reply:
``AuthorityTests.ThrowIfNotSupportedForMtls_ForInvalidAuthorityType_ThrowsException`` covers them.